desktop/hardened-desktop/admin-controls/configure-ac.md

```diff
@@ -0,0 +1,129 @@
+- Windows: /ProgramData/DockerDesktop/admin-settings.json
+- Linux - /usr/share/docker-desktop/admin-settings.json 
+
+By placing this file in the above protected directories, end users are unable to modify it.
+
+>Note
+>
+> It is assumed that you have the ability to push the `admin-settings.json` settings file to the locations specified above through a device management software such as [Jamf](https://www.jamf.com/lp/en-gb/apple-mobile-device-management-mdm-jamf-shared/?attr=google_ads-brand-search-shared&gclid=CjwKCAjw1ICZBhAzEiwAFfvFhEXjayUAi8FHHv1JJitFPb47C_q_RCySTmF86twF1qJc_6GST-YDmhoCuJsQAvD_BwE).
+
+## Step two: Add the key value pairs for the admin controls you want to lock in
+
+>Note
+>
+>Some of the configuration parameters only apply to Windows. This is highlighted in the table below.
+
+The `admin-settings.json` file requires a nested list of configuration parameters, each of which must contain the  `locked` parameter. 
+If set to `true`, users are not able to edit this setting from Docker Desktop or the CLI. 
+If set to `false`, users can change this setting from Docker Desktop or the CLI by directly editing the `settings.json` file. If this setting is omitted, the default value is `false`.
+
+The following `admin-settings.json` code and table provides the required syntax and descriptions for parameters and values:
+
+```json
+{
+  "configurationFileVersion": 2,
+  "exposeDockerAPIOnTCP2375": {
+    "locked": true,
+    "value": false
+  },
+  "proxy": {
+    "locked": false,
+    "mode": "system",
+    "server": "myproxy.com",
+    "port":3129,
+    "exclude": ["foo.com", "bar.com"]
+  },
+  "enhancedContainerIsolation": {
+    "locked": false,
+    "value": false
+  },
+  "useWindowsContainers": {
+      "locked": false,
+      "value": false
+  },
+  "linuxVM": {
+    "wslEngineEnabled": {
+      "locked": false,
+      "value": false
+    },
+    "dockerDaemonOptions": {
+      "locked": false,
+      "value":"<json string>"
+    },
+    "vpnkitCIDR": {
+      "locked": false,
+      "value":"192.168.65.0/24"
+    }
+  },
+  "windowsContainers": {
+    "dockerDaemonOptions": {
+      "locked": false,
+      "value":"<json string>"
+    }
+  },
+  "disableUpdate": {
+    "locked": false,
+    "value": false
+  },
+  "analyticsEnabled": {
+    "locked": false,
+    "value": true
+  },
+}
+```
+
+| Parameter                        | Description                      |
+| :------------------------------- | :------------------------------- |
+| `configurationFileVersion`        | Specifies the version of the configuration file format.    |
+| `exposeDockerAPIOnTCP2375` |<span class="badge badge-info">Windows only</span> Exposes the Docker API on a specified port. If `value` is set to true, the Docker API is exposed on port 2375. Note: This is unauthenticated and should only be enabled if protected by suitable firewall rules.|
+| `proxy` | It is used for `http` and `https`.  If the port is custom, specify it in the property. |
```

Comment:
for keys like proxy, which take an object which has more keys, like "locked", "server", "port" etc, we should mention those things.

for each key throughout this doc that is not a boolean, we should mention the specific values that are accepted.